genai-function-calling/openai-agents/README.md

```diff
@@ -52,6 +52,22 @@ Finally, run `main.py` (notice the prefix of `opentelemetry-instrument):
 dotenv run --no-override -- opentelemetry-instrument python main.py
 ```
 
+## Run with Model Context Protocol (MCP)
+
+[mcp_server](mcp_server.py) includes code needed to decouple tool discovery and invocation
+via the [Model Context Protocol (MCP) flow][flow-mcp]. To run using MCP, append
+`-- --mcp` flag to `dotenv run` or `docker compose run` command.
+
+For example, to run with Docker:
+```bash
+docker compose run --build --rm genai-function-calling -- --mcp
```

Comment:
I think..

```suggestion
docker compose run --build --rm genai-function-calling --mcp
```

Reply:
Oops yeah just copied this from vercel-ai without testing it. I think it may need the `CMD` -> `ENTRYPOINT` change